recipes-ni/niacctbase/niacctbase.bb

```diff
@@ -1,4 +1,7 @@
-DESCRIPTION = "Basic user and group settings for NI Linux RT applications."
+SUMMARY = "Common user and group settings for NI applications"
```

Comment:
Should it be `applications and devices` based on your most recent changes?

Reply:
Done.